practice/postgres/queries/003-data-models.sql

```
@@ -0,0 +1,19 @@
+-- View all courses
+SELECT * FROM courses;
+
+-- View enrollments
+SELECT * FROM enrollments;
+
+-- Demonstrate relationships
+SELECT 
+    s.name AS student, 
+    c.title AS course 
+FROM enrollments e
+JOIN students e
```

SQLFluff (4.2.2)

[error] Duplicate table alias 'e'. Table aliases should be unique.

(AL04)